api/app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Types
+  class UserType < Types::BaseObject
+    field :id, ID, null: false
+    field :email, String, null: false
+    field :full_name, String, null: false
+    field :user_tasks, [Types::TaskType], null: true
+
+    def full_name
+      # `object` references the user instance
+      [object.first_name, object.last_name].compact.join(' ')
+    end
+
+    def user_tasks
+      @tasks = Task.where(user_id: id)
```

Comment:
See my comment above.

But in case you did want to call this something other than tasks, you could still use `object.tasks` instead of `Task.where(user_id: id)`. `object` is the task in this case. You also don't need to add the `@tasks =` here. This method just needs to return a list of tasks.